fedimint-ui/packages/translation/locales/en/guardian.json

```diff
@@ -0,0 +1,119 @@
+{	
+	"app": {
+		"error": "Something went wrong."
+	},
+	"connect_guardians": {
+		"invite_guardians": "Invite Followers",
+		"invite_guardians_help": "Share this link with the other Guardians",
+		"approve": "Approve",
+		"approved": "Approved",
+		"pedning": "Pending",
+		"not_joined": "Not joined"
+	},
+	"federation_dashboard": {
+		"card_header": "Federation info",
+		"your_status": "Your status",
+		"epoch_count": "Epoch count",
+		"api_version": "API Version",
+		"consensus_version": "Consensus version",
+		"peer_info": "Peer info",
+		"name": "Name",
+		"status": "Status",
+		"peer": "Peer"
+	},
+	"login": {
+		"password": "Password",
+		"submit": "Submit"
+	},
+	"role_selector": {
+		"Leader": {
+			"label": "Leader",
+			"description": "Choose one of your Guardians as a Leader. The Leader will input information about the Federation."
+		},
+		"Follower": {
+			"label": "Follower",
+			"description": "Guardian Followers (all other Guardians) will confirm information that the Leader inputs."
+		},
+		"next": "Next"
+	},
+	"run_dkg": {
+		"error_config": "Failed to run distributed key generation. Federation setup must be restarted.",
+		"error_default": "Not ready for DKG, your current status is",
+		"error_header": "Something went wrong.",
+		"waiting_header": "Waiting for peers...",
+		"generating_header": "Generating codes..."
+	},
+	"set_config": {
+		"guardian_name": "Guardian name",
+		"guardian_name_help": "This name will be shown to other Guardians",
+		"admin_password": "Admin password",
+		"admin_password_help": "You’ll need this every time you visit this page.",
```

Comment:
My IDE shows this for this line. Don't think it's an issue but we should probably use en characters for the `en` files. 
<img width="763" alt="Screenshot 2023-05-19 at 7 24 20 AM" src="https://github.com/fedimint/fedimint/assets/133234413/4be7f8d4-30da-421b-be23-7d0c6e521105">